agm.html

```diff
@@ -0,0 +1,78 @@
+<!--#set var="title" value="AGM 2021" -->
+<!--#include virtual="inc/header.html" -->
+<h2>SRCF AGM 2021</h2>
+
+<div class="column">
+  <div class="row">
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="card">
+        <div class="card-body">
+          <h4 class="card-title">Meeting details</h4>
+          <p>The 2021 Annual General Meeting of the Student-Run Computing Facility
+            will be held during the final week of Lent term.</p>
+          <p>The meeting will commence at 14:00 on Saturday 20 March 2021.</p>
```

Comment:
I wouldn't usually specify a timezone, but given everyone's distributed nature this year it might be worth saying 'GMT'.